### PR TITLE
[serdes] Bring back the big comments on the serdes process

### PR DESCRIPTION
**Pull request by bshepherdson**

### CODE REVIEW

**Comment by bshepherdson** [comment]
Current dependencies on/for this PR:
* master
  * **PR #35468** <a href="https://app.graphite.dev/github/pr/metabase/metabase/35468" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/metabase/metabase/35468?utm_source=stack-comment).

File: src/metabase/models/serialization.clj [hunk 1]
```diff
@@ -981,6 +1218,7 @@
          (filter some?)
          (reduce set/union #{}))))
 
+;;; <h2>Memoizing appdb lookups</h2>
```
Comment (piranha) [inline src/metabase/models/serialization.clj:1221]: I want Markdown here! 😁

**Review by sansolovyov — APPROVED** [review]
Cool!

File: src/metabase/models/serialization.clj (hunk 1 above)
Comment (piranha) [reply src/metabase/models/serialization.clj:1221]: Oh, turns out it's ok to use markdown here... Hope you won't mind if I change headers to use `## `. :)

**Pull request merged**